jdk/make/closed/autoconf/custom-spec.gmk.in

```diff
@@ -61,6 +61,7 @@ OPENJ9_ENABLE_DEMOS                 := @OPENJ9_ENABLE_DEMOS@
 OPENJ9_ENABLE_JFR                   := @OPENJ9_ENABLE_JFR@
 OPENJ9_ENABLE_JITSERVER             := @OPENJ9_ENABLE_JITSERVER@
 OPENJ9_ENABLE_OPENJDK_METHODHANDLES := @OPENJ9_ENABLE_OPENJDK_METHODHANDLES@
+OPENJ9_ENABLE_SNAPSHOTS		    := @OPENJ9_ENABLE_SNAPSHOTS@
```

Comment:
Tabs should only be used for indentation (not after any non-whitespace character).